kind/kind-up-test.sh

```diff
@@ -24,8 +18,8 @@ nodes:
     protocol: TCP
```

Comment:
@bouskaJ and here, too, can you change `hostPort: 8888` and `hostPort: 4433`? for rootless